R/input-time-series.R

```diff
@@ -0,0 +1,186 @@
+##' Prepare data for ART input time series plots
+##'
+##' Take uploaded ART and shape file paths and format as data which
+##' can be used to draw input time series graphs.
+##'
+##' @param art Path to file containing ART data
+##' @param shape Path to file containing geojson areas data
+##'
+##' @return Data formatted for plotting input time series containing columns
+##' area_id, area_name, area_level_label, time_step, time_period, plot and
+##' value
+##' @export
+prepare_input_time_series_art <- function(art, shape) {
+
+  ## From "area_id", "calendar_quarter", "sex", "age_group", "art_current"
+
+  ## To "area_id","area_name","area_level_label", "area_level", time_step",
+  ## "time_period","plot", "value"
+
```

Comment:
```suggestion
```

Reply:
These were just notes I made when there was no impl here but I think not useful anymore.